mikupad.html

```diff
@@ -2324,6 +2342,19 @@
 			case 'false:false:false:Escape':
 				cancel();
 				break;
+			case 'false:true:false:r':
+        	case 'false:false:true:r':
+				undoAndPredict();
+            	break;
+			case 'false:true:false:z':
+        	case 'false:false:true:z':
+				undo();
+            	break;
+			case 'false:true:false:y':

```

Comment:
It's worth to also support CTRL+SHIFT+Z, which is also a standard for redo.
```suggestion
			case 'false:true:true:Z':
			case 'false:true:false:y':
```